Engine/Source/Editor/UnrealEd/Private/CookOnTheFlyServer.cpp

```diff
@@ -5594,7 +5595,16 @@ void UCookOnTheFlyServer::EmptyPackageLists()
 	// We empty these lists because otherwise too many items are cooked
 	PackageTracker->LoadedPackages.Empty();
 	PackageTracker->NewPackages.Empty();
+
+	// We don't to want startup packages
```

Comment:
Can you put in the comment the **reason why** we don't want startup packages?

Reply:
done.